probeinterface/io.py

```diff
@@ -139,6 +139,58 @@ def read_prb(file):
     return probegroup
 
 
+def read_maxwell(file):
+    """
+    Read a maxwell file and return a ProbeGroup object.
+
+    Since Maxwell do not handle electrode shape then circle of 5um are put.
+    Same for electrode shape a dummy tip is put.
+
+    Maxwell format do not contain any information about the channel of the probe
+    Only the channel index on device is given.
+
+    """
+
+    file = Path(file).absolute()
+    assert file.is_file()
+
+    try:
+        import h5py
+    except ImportError as error:
+        print(error.__class__.__name__ + ": " + error.message)
+
+
+    my_file = h5py.File(file, mode='r')
```

Comment:
@yger note that for maxtwo data the path is different. This should be ready maxone or handle both cases IMO

Reply:
This is from the script I use to handle maxtwo data. What is different in maxone? I think both should work

Reply:
So for the data I have seen, multiple recordings can be saved in the same file (e.g. when doing an activity scan). In that case, each recording has a different config. The `mapping` is at this path:

```
f["wells"]["well00i"]["rec000j"]["mapping"]
```

where ["well00i"]["rec000j"] indicates the i-th well and j-th recording.

Reply:
See `MaxTwoRecordingExtractor`here: https://github.com/SpikeInterface/spikeextractors/blob/maxtwo/spikeextractors/extractors/maxwellextractors/maxwellextractors.py